wp-admin/js/widgets/media-widgets.js

```diff
@@ -152,9 +153,13 @@ wp.mediaWidgets = ( function( $ ) {
 				attachment = new wp.media.model.Attachment( {
 					id: control.model.get( 'attachment_id' )
 				} );
-				attachment.fetch().done( function() {
-					control.selectedAttachment.set( attachment.attributes );
-				} );
+				attachment.fetch()
+					.done( function() {
+						control.selectedAttachment.set( attachment.attributes );
+					} )
+					.fail( function() {
+						control.selectedAttachment.set( { error: true } );
```

Comment:
Maybe the `error` here should be a code like `missing_attachment` instead of a boolean to anticipate additional error types in the future.